flake.nix

```diff
@@ -0,0 +1,20 @@
+{
+  description = "raw photography workflow that sucks less";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, utils, ... }:
+    utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+      in
+      {
+        devShell = with pkgs; mkShell {
+          inputsFrom = [ pkgs.vkdt ];
```

Comment:
If this flake is to target the development branches of vkdt, we should really list out all of the dependencies of vkdt and not rely on the package from nixpkgs. the package in nixpkgs only gets updated when there is a release. This will also have the effect of making it easier to do the `nix run ...` part of the flake as well.

Reply:
fair enough